build_posix/configure.ac.in

```diff
@@ -29,18 +29,22 @@ define([AC_LIBTOOL_LANG_F77_CONFIG], [:])dnl
 # reason to believe "c++" can build compatible objects.
 #
 # Check whether the C++ compiler works by linking a trivial program.
-if test "$CC" = "cc"; then
-	AC_CACHE_CHECK([whether the C++ compiler works],
-		       [wt_cv_prog_cxx_works],
-		       [AC_LANG_PUSH([C++])
-			AC_LINK_IFELSE([AC_LANG_PROGRAM([], [])],
-				       [wt_cv_prog_cxx_works=yes],
-				       [wt_cv_prog_cxx_works=no])
-			AC_LANG_POP([C++])])
-else
-	AC_MSG_WARN([C++ compiler ignored unless compiler is named "cc"])
-	wt_cv_prog_cxx_works=no
-fi
+# Check if CC is set as "cc"
+AM_CONDITIONAL([IS_CPP], [test $CC = "cc"])
+# If IS_CPP is true, don't do anything, otherwise try to compare g++ and gcc versions
```

Comment:
Can we use a more descriptive variable name than `IS_CPP`?  I read that as "is it c++, or is it cpp" (cpp is a different command in linux) Maybe `IS_CXX_OKAY`?   The comment should not read the code.